README.md

```diff
@@ -15,6 +15,11 @@ on how to do that, including how to develop and test locally and the versioning
 
 _Note: 1.28.0 and later require Gradle 7_
 
+### TBD
+*Released*: TBD
```

Comment:
Since there are no other changes pending or planned, I'll release this once this branch is merged, so please update these values to 1.34.0 and today's date before merging this branch (to save an additional merge).